llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -3193,6 +3239,9 @@ class LLVM_ABI_FOR_TEST VPWidenMemoryRecipe : public VPRecipeBase,
   /// Whether the consecutive accessed addresses are in reverse order.
   bool Reverse;
 
+  /// Whether the consecutive accessed addresses are compressed with mask value.
+  bool Compressed;
+
```

Comment:
This corresponds 1-1 to an intrinsic, right? Can we just use VPWidenIntrinsicRecipe?